### ISSUE TITLE
Clicking compiled script entry in "loaded scripts" loads the script from the runtime, not the same script on disk

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#62183]
Testing #62093

- Debug anything that includes a .js script on disk
- Click the '.js' script in the loaded scripts view
- See that it opens a copy of that script with content loaded from node (with the IIFE module wrapper) instead of the version on disk

Is this by design? Seems like it should open the version on disk

![image](https://user-images.githubusercontent.com/323878/47740733-c1f16400-dc35-11e8-82cc-0262a9fff318.png)

**Comment by weinand** [comment]
yes, that's by design. The loaded scripts view always shows the "truth".
